### PR TITLE
kupfer: fix broken build

### PR DESCRIPTION
**Pull request by aanderse**
###### Motivation for this change
https://github.com/NixOS/nixpkgs/issues/60498

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/kupfer/default.nix [hunk 1]
```diff
@@ -14,12 +14,14 @@
 with python3Packages;
 
 buildPythonApplication rec {
-  name = "kupfer-${version}";
+  pname = "kupfer";
   version = "319";
 
-  src = fetchurl {
-    url = "https://github.com/kupferlauncher/kupfer/releases/download/v${version}/kupfer-v${version}.tar.xz";
```
Comment (jtojnar) [inline pkgs/applications/misc/kupfer/default.nix]: This is a dist tarball, not a repo snapshot. Will this not break the application version information?

Comment (aanderse) [reply pkgs/applications/misc/kupfer/default.nix]: You are correct. My mistake. Thank you for catching.

**Comment by aanderse** [comment]
This also needs backport as it is currently broken in 19.03.

File: pkgs/applications/misc/kupfer/default.nix [hunk 2]
```diff
@@ -30,6 +30,8 @@ buildPythonApplication rec {
   buildInputs = [ hicolor-icon-theme docutils libwnck3 keybinder3 ];
   propagatedBuildInputs = [ pygobject3 gtk3 pyxdg dbus-python pycairo ];
 
+  strictDeps = false;
```
Comment (jtojnar) [inline pkgs/applications/misc/kupfer/default.nix]: Could you also add a comment with explanation, pointing to https://github.com/NixOS/nixpkgs/issues/56943

Comment (aanderse) [reply pkgs/applications/misc/kupfer/default.nix]: I hope the comment I have added is sufficient.

**Review by jtojnar — APPROVED** [review]
lgtm

**Pull request merged**